### PR TITLE
Feature/chapter 2 items

### PR DESCRIPTION
**Pull request by richardbuckle**
Conformed item definitions to FD's spreadsheet and added new ones.

### CODE REVIEW

File: DataDefinitions/Material.cs [hunk 1]
```diff
@@ -102,6 +102,9 @@ static Material()
             var Tg_ShipSystemData = new Material("tg_shipsystemsdata", Data, Rare);
             var Tg_ResidueData = new Material("tg_residuedata", Data, Rare);
             var UnknownWakeScan = new Material("unknownwakedata", Data, Rare);
+            var Guardian_VesselBlueprint = new Material("guardian_vesselblueprint", Data, Rare);
```
Comment (Tkael) [inline DataDefinitions/Material.cs]: I may be wrong, but I don't think that `Guardian_VesselBlueprint` appears in Howard's spreadsheet.
Per the in-game HUD display, all 3 of these are very rare. But my capacity for Guardian Vessel prints is 100 while my capacity for the other two is 150. Thus, I think we should classify Guardian Vessel prints as "Very Rare" (with an auto-generated max capacity of 100 units) while classifying the other two as "Rare" (with an auto-generated max capacity of 150 units).

Comment (richardbuckle) [reply DataDefinitions/Material.cs]: I’ll check on Guardian_VesselBlueprint and conform the rarities and capacities.

**Comment by Tkael** [comment]
We may also want to add things like cockpits. Here's data for the Krait:
```
Add Krait MkII cockpit?
2018-06-29T04:21:56 [Info] ResourceBasedLocalizedEDName:FromEDName Unknown ED name krait_mkii_cockpit in resource EddiDataDefinitions.Properties.Modules 
2018-06-29T04:21:56 [Info] JournalMonitor:ParseJournalEntry Unknown module krait_mkii_cockpit {"Slot":"ShipCockpit","Item":"krait_mkii_cockpit","On":true,"Priority":1,"Health":1.0}
```

File: DataDefinitions/Material.cs (hunk 1 above)
Comment (richardbuckle) [reply DataDefinitions/Material.cs]: I agree. Done.

**Pull request merged**